### ISSUE TITLE
New version bug: raise RuntimeError("location not created nor specified")

### ISSUE THREAD

**Issue by cyber778** [pypa/pipenv#5775]
Hi, I'm using docker and when I get to this command: `RUN pipenv install --system --ignore-pipfile`
I see:
```
 ---> Running in 8b72dbd2a17c
Traceback (most recent call last):
  File "/usr/local/bin/pipenv", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.8/site-packages/pipenv/vendor/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/pipenv/cli/options.py", line 58, in main
    return super().main(*args, **kwargs, windows_expand_args=False)
  File "/usr/local/lib/python3.8/site-packages/pipenv/vendor/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/pipenv/vendor/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/pipenv/vendor/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/pipenv/vendor/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/pipenv/vendor/click/decorators.py", line 84, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/pipenv/vendor/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/pipenv/cli/command.py", line 233, in install
    do_install(
  File "/usr/local/lib/python3.8/site-packages/pipenv/routines/install.py", line 66, in do_install
    ensure_project(
  File "/usr/local/lib/python3.8/site-packages/pipenv/utils/project.py", line 82, in ensure_project
    os.environ["PIP_PYTHON_PATH"] = project.python
  File "/usr/local/lib/python3.8/site-packages/pipenv/project.py", line 1163, in python
    return project_python(self)
  File "/usr/local/lib/python3.8/site-packages/pipenv/utils/shell.py", line 404, in project_python
    python = project._which("python")
  File "/usr/local/lib/python3.8/site-packages/pipenv/project.py", line 1172, in _which
    raise RuntimeError("location not created nor specified")
RuntimeError: location not created nor specified
```

This is a new bug since yesterday (latest version release) and it is killing all my ENVs.
My fix was to run: `pip install pipenv==2023.6.26`

As I'm working on a confidential project I can't share my pipfile but I'm sure this will pop up with others soon enough.

Keep up the good work!

**Comment by matteius** [comment]
Feel free to open a PR to fix this!

**Comment by matteius** [comment]
Actually, have you tried `2023.7.3`?  I can't reproduce this ... and it seems a lot like the bug that was reported yesterday #5772

**Comment by matteius** [comment]
I am closing this as the duplicate issue report as I believe it was fixed in `2023.7.3`
